graphql/src/main/java/com/linecorp/armeria/common/graphql/Graphql.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.graphql;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import graphql.GraphQLContext;
+
+/**
+ * Utility for handling the GraphQL.
+ */
+@UnstableApi
+public final class Graphql {
+
+    private static final String GRAPHQL_CONTEXT_KEY = "com.linecorp.armeria";
```

Comment:
Should we set a more specific name?
```suggestion
    private static final String GRAPHQL_CONTEXT_KEY = "com.linecorp.armeria.graphql.context.key";
```